server/lib/pbench/process_tb.py

```diff
@@ -95,9 +99,13 @@ def process_tb(self) -> Results:
             tb = Path(str(tbmd5)[0 : -len(".md5")])
             tbdir = tb.parent
             controller = tbdir.name
+            satellite_prefix = None
```

Comment:
Technically, if you're passing `None` then the method type hint is wrong. It *should* have the `Optional[str]` notation, because `None` isn't a `str`. However, that doesn't make it legal to omit the parameter entirely, because there's no default value. Given the code here, that doesn't matter since you're always passing it, but it should still be `Optional[str]`.

(Python doesn't care: to Python, all parameters are `Any`, regardless of type hint, and that includes `None`. The current code should cause it to be flagged in IDEs... although since this is 0.69 that's probably not a big deal.)

Reply:
I second this.